com.unity.render-pipelines.universal/Runtime/RendererFeatures/ScreenSpaceShadows.cs

```diff
@@ -0,0 +1,214 @@
+                m_RenderTextureDescriptor.msaaSamples = 1;
+                m_RenderTextureDescriptor.graphicsFormat = RenderingUtils.SupportsGraphicsFormat(GraphicsFormat.R8_UNorm, FormatUsage.Linear | FormatUsage.Render)
+                    ? GraphicsFormat.R8_UNorm
+                    : GraphicsFormat.B8G8R8A8_UNorm;
+
+                cmd.GetTemporaryRT(m_RenderTarget.id, m_RenderTextureDescriptor, FilterMode.Point);
+
+                RenderTargetIdentifier renderTargetTexture = m_RenderTarget.Identifier();
+                ConfigureTarget(renderTargetTexture);
+                ConfigureClear(ClearFlag.None, Color.white);
+            }
+
+            /// <inheritdoc/>
+            public override void Execute(ScriptableRenderContext context, ref RenderingData renderingData)
+            {
+                if (material == null)
+                {
+                    Debug.LogErrorFormat("{0}.Execute(): Missing material. {1} render pass will not execute. Check for missing reference in the renderer resources.", GetType().Name, profilerTag);
+                    return;
+                }
+
+                Camera camera = renderingData.cameraData.camera;
+
+                CommandBuffer cmd = CommandBufferPool.Get();
+                using (new ProfilingScope(cmd, ProfilingSampler.Get(URPProfileId.ResolveShadows)))
+                {
+                    if (!renderingData.cameraData.xr.enabled)
+                    {
+                        cmd.SetViewProjectionMatrices(Matrix4x4.identity, Matrix4x4.identity);
+                        cmd.DrawMesh(RenderingUtils.fullscreenMesh, Matrix4x4.identity, material);
+                        cmd.SetViewProjectionMatrices(camera.worldToCameraMatrix, camera.projectionMatrix);
+                    }
+                    else
+                    {
+                        // Avoid setting and restoring camera view and projection matrices when in stereo.
+                        RenderTargetIdentifier screenSpaceShadowTexture = m_RenderTarget.Identifier();
+                        cmd.Blit(null, screenSpaceShadowTexture, material);
+                    }
+
+                    CoreUtils.SetKeyword(cmd, ShaderKeywordStrings.MainLightShadows, false);
+                    CoreUtils.SetKeyword(cmd, ShaderKeywordStrings.MainLightShadowCascades, false);
+                    CoreUtils.SetKeyword(cmd, ShaderKeywordStrings.MainLightShadowScreen, true);
+                }
+
+                context.ExecuteCommandBuffer(cmd);
+                CommandBufferPool.Release(cmd);
+            }
+
+            /// <inheritdoc/>
+            public override void OnCameraCleanup(CommandBuffer cmd)
+            {
+                if (cmd == null)
+                {
+                    throw new ArgumentNullException("cmd");
+                }
+
+                cmd.ReleaseTemporaryRT(m_RenderTarget.id);
+            }
+        }
+
+        private class RestoreShadowKeywordsPass : ScriptableRenderPass
```

Comment:
You can add a call to ConfigureTarget(BuitinRenderTextureType.Current) to avoid this pass changing the render pass to camera by default.

Reply:
Applied it :)